Adding initial commits of ps/2 converter (#5730)
qmk/qmk_firmware: keyboards/converter/ps2_usb/README.md

> PS/2 to USB keyboard converter

Reviewer: Could you please rename this file as `readme.md`?

Author: yup

Reviewer: This file still needs to be renamed.

==============================
This firmware converts PS/2 keyboard protocol to USB.(It supports Scan Code Set 2.)

Connect Wires
-------------
1. Connect **Vcc** and **GND**.
2. Connect **Clock** and **Data** line. 
    - **Interrupt**:   **Clock** is on `PD1` and **Data** on `PD0`.(Recommended. Soarer's converter compatible)
    - **Busywait**:    **Clock** is on `PD1` and **Data** on `PD0`.
    - **USART**:       **Clock** is on `PD5` and **Data** on `PD2`.
3. You need pull-up resistors. 1K-10K Ohm would be fine.

> To change pin configuration edit **config.h** and **Makefile**.

Reviewer 2: ```suggestion
To change pin configuration edit **config.h** and **rules.mk**.
```

Build Firmware
--------------

> For **TMK converter Rev.2**:

Reviewer 2: ```suggestion
For **QMK converter**:
```

>     $ make -f Makefile.rev2 clean

Reviewer 2: This isn't needed, technically
```suggestion
```

>     $ make -f Makefile.rev2 KEYMAP=plain

Reviewer 2: ```suggestion
    $ make converter/ps2_usb:default
```

Or even:
```suggestion
    $ make distclean converter/ps2_usb:default
```

To program firmware push the button on converter and run:

>     $ make -f Makefile.rev2 KEYMAP=plain dfu

Reviewer 2: ```suggestion
    $ make converter/ps2_usb:default:dfu
```

but if this is a teensy controller (eg, halfkay), then it should be:
```suggestion
    $ make converter/ps2_usb:default:teensy
```

> - For **TMK converter Rev.1** use `make -f Makefile.rev1` instead.

Reviewer 2: ```suggestion
```

> - To select keymap use `jis`, `spacefn` or your own in place of `plain`.

Reviewer 2: ```suggestion
```

Keymap
------

> Several version of keymap are available in advance but you are recommended to define your favorite layout yourself. To define your own keymap create file named `keymap_<name>.c` and see keymap document(you can find in README.md of top directory) and existent keymap files.

Reviewer 2: ```suggestion
Several version of keymap are available in advance but you are recommended to define your favorite layout yourself. To define your own keymap create a folder with your github username, and in it, a file named `keymap.c` and see keymap document(you can find in README.md of top directory) and existent keymap files.
```
